modules/ROOT/pages/troubleshooting-visualization.adoc

```diff
@@ -0,0 +1,86 @@
+= Troubleshooting Visualization
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+3
+
+== About the Troubleshooting Visualization
+
+The Troubleshooting Visualization shows you the current and past health of your Mule apps and APIs. It allows you to quickly visualize where your network is slow, receiving heavy traffic or failing. From this visualization, you can drill down directly into Anypoint Monitoring dashboards or logs to continue to troubleshoot the performance of your systems.
```

Comment:
Let's get rid of "and APIs" in the first sentence.

Reply:
```suggestion
The Troubleshooting Visualization shows you the current and past health of your Mule apps. It allows you to quickly visualize where your network is slow, receiving heavy traffic or failing. From this visualization, you can drill down directly into Anypoint Monitoring dashboards or logs to continue to troubleshoot the performance of your systems.
```